src/middleware/compress/index.test.ts

```diff
@@ -0,0 +1,62 @@
+import { Hono } from '../../hono'
+import { compress } from '.'
+
+describe('Parse Compress Middleware', () => {
+  
+  if (parseInt(process.versions.node.split('.')[0]) < 17) {
```

Comment:
@tani 

I think this version check is not needed. We don't use node `node:stream/web`.

Reply:
Noted. Thanks